packages/salesforcedx-vscode-core/src/commands/forceSourceStatus.ts

```diff
@@ -20,22 +21,28 @@ import {
 
 export enum SourceStatusFlags {
   Local = '--local',
-  Remote = '--remote'
+  Remote = '--remote',
 }
 
 export class ForceSourceStatusExecutor extends SfdxCommandletExecutor<{}> {
+  public command: string;
   private flag: SourceStatusFlags | undefined;
+  private description: string;
+  private logName: string;
 
-  public constructor(flag?: SourceStatusFlags) {
+  public constructor(flag?: SourceStatusFlags, cmdVersion?: CommandVersion) {
     super();
     this.flag = flag;
+    this.command = `force:source:${cmdVersion? cmdVersion : CommandVersion.Default}:status`;
+    this.description = ['force_source_status', cmdVersion, 'text'].filter(Boolean).join('_');
```

Comment:
if we're worried about it not being defined on line 36 should we not also have the same concern here?  maybe assign it to a variable and use the variable in both places?

Reply:
Current refactored code doesn't use this.